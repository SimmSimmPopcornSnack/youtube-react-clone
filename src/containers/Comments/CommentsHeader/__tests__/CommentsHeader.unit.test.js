import React from "react";
import { shallow } from "enzyme";
import { CommentsHeader } from "../CommentsHeader";

describe("CommentsHeader", () => {
    test("renders without props", () => {
        const wrapper = shallow(<CommentsHeader/>);
        expect(wrapper).toMatchSnapshot();
    });
    test("renders with amountComments", () => {
        const wrapper = shallow(
            <CommentsHeader amountComments={112499}/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
