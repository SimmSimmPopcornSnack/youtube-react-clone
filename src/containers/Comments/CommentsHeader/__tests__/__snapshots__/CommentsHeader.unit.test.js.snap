// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader renders with amountComments 1`] = `
<div
  className="comments-header"
>
  <h4>
    112499
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;

exports[`CommentsHeader renders without props 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;
